README.md

```diff
@@ -60,6 +66,57 @@ cp envs/env_vars_deactivate.sh <conda_prefix>/etc/conda/deactivate.d/env_vars.sh
 If necessary, create any missing directories manually.
 In case some lines in the environment scripts cause problems, you can edit the files to trouble-shoot.
 
+These operations are also automated in the `envs/set_vars.sh` script, which you can call via
+
+```console
+. envs/set_vars.sh <conda_prefix>
+```
```

Comment:
mention above that there are 2 ways to do this. And then mention the script first... as most ppl will probably go for the easier option.